client/src/components/Sub_RelatedItems/SingleProd.jsx

```diff
@@ -81,11 +81,9 @@ export default function SingleProd({
       totalRatings += Number(ratings[num]);
     }
 
-    return [(sum / totalRatings).toFixed(2), totalRatings];
-  };
-  // =================== HANDLERS ===================
-  const handleOutsideClick = () => {
-    return null;
+    const thisRoundedRating = Math.round((sum / totalRatings).toFixed(2) / 0.25) * 0.25;
+
+    return [thisRoundedRating, totalRatings];
```

Comment:
can I ask you why you changed this?

Reply:
The BRD states that the stars should be filled in to the nearest quarter percent. So, I converted each avg. rating into the nearest .0, .25, .50, or .75.

Reply:
gotcha